### PR TITLE
Feature/1078 extend provider loader with itemrunner

### PR DESCRIPTION
**Pull request by btamas**
Related to https://oat-sa.atlassian.net/browse/NEX-1078

- The nextgen testrunner configuration is provided by the deliver backend. This config contains provider list like test runner, communicator etc. The new UI test runner won't load the new UI item runner automatically, so it should be triggered. I created a new configuration key, called `itemRunner` and I extended the provider loader to be able to load it.
- I've mocked the item runner, so test runner still doesn't depend on item runner.
- I've written small test for the modifications.
- From now, the test runner configuration will contain HTMLElement container. https://github.com/oat-sa/tao-test-runner-fe/blob/fix/1078_extend-providerLoader-with-itemrunner/src/runnerComponent.js#L135
- I had to upgrade package versions, because there were a warning regarding outdated browserlist

- [ ] Do not forget to release it after merge

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -40,9 +40,9 @@
     },
     "homepage": "https://github.com/oat-sa/tao-test-runner-fe#readme",
     "devDependencies": {
-        "@babel/core": "^7.4.5",
-        "@babel/preset-env": "^7.4.5",
-        "@oat-sa/browserslist-config-tao": "^0.1.0",
+        "@babel/core": "^7.10.1",
+        "@babel/preset-env": "^7.10.1",
+        "@oat-sa/browserslist-config-tao": "^0.1.1",
```
Comment (krampstudio) [inline package.json:45]: you need to add the dependency to `@oat-sa/tao-item-runner`

**Comment by krampstudio** [comment]
- [x] New code is covered by tests (if applicable)
- [x] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful

File: src/runnerComponent.js [hunk 2]
```diff
@@ -132,7 +132,7 @@ export default function runnerComponentFactory(container = null, config = {}, te
         })
         .on('render', function() {
             const runnerConfig = Object.assign(_.omit(this.config, ['providers']), {
-                renderTo: this.getElement()
+                renderTo: this.getElement().get(0)
```
Comment (krampstudio) [inline src/runnerComponent.js]: it looks like there are still some external expectation for it to be a jquery element :/

**Review by krampstudio — CHANGES_REQUESTED** [review]

**Review by shaveko — APPROVED** [review]
Code looks ok and test run doesn't return errors

**Review by krampstudio — APPROVED** [review]
Now previews are working as expected :+1:

**Pull request merged**